src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/NumberToLocalizedStringTransformerTest.php

```diff
@@ -647,6 +661,59 @@ public function testReverseTransformENotation($output, $input)
         $this->assertSame($output, $transformer->reverseTransform($input));
     }
 
+    public function testReverseTransformWrapsIntlErrorsWithErrorLevel()
+    {
+        if (!\extension_loaded('intl')) {
+            $this->markTestSkipped('intl extension is not loaded');
```

Comment:
You can use `@requires extension intl` instead

Reply:
I took this from the existing tests in DateTimeToLocalizedStringTransformerTest. Should I also convert it there?

Reply:
yes please